scripts/fuzz_opt.py

```diff
@@ -207,6 +207,7 @@ def init_important_initial_contents():
         os.path.join('lit', 'passes', 'optimize-instructions-call_ref.wast'),
         os.path.join('lit', 'passes', 'inlining_splitting.wast'),
         os.path.join('heap-types.wast'),
+        os.path.join('spec', 'ref_cast.wast'),
```

Comment:
Intentional?

(you can just set RECENT_DAYS to 1 or 2, btw)

Reply:
This was intentional, but I didn't know about RECENT_DAYS. I'll revert and use that instead. Meanwhile I found a place where OptimizeInstructions needs to be updated, so I'll push an update for that.